lib/Conversion/ImportVerilog/CMakeLists.txt

```diff
@@ -15,15 +13,10 @@ add_compile_definitions($<$<CONFIG:Debug>:DEBUG>)
 if (MSVC)
   # No idea what to put here
 else ()
```

Comment:
Can you not specify slang include directories as being -isystem so that you don't need these warning suppressions?